### PR TITLE
Add pylint error check

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/lib/filter_capabilities.py [hunk 1]
```diff
@@ -156,7 +156,9 @@ def caching_prepare_input_source(source, base=None):
             args = (source,) if base is None else (source, base)
             return old_prepare_input_source(*args)
 
-        document = urlopen(full_uri, proxies=proxies)
+        opener = urllib.request.FancyURLopener(proxies)
+        with opener.open(full_uri) as stream:
+            document = stream.read()
```
Comment (gberaudo) [inline geoportal/c2cgeoportal_geoportal/lib/filter_capabilities.py]: For my information, why this change?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/lib/filter_capabilities.py]: The old one don't works with Python3, reported by PyLint
